website/docs/concepts/combining_providers.mdx

```diff
@@ -268,6 +267,25 @@ final productsProvider = FutureProvider<List<Product>>((ref) async {
 });
 ```
 
+You can also use `.select` in simple cases which will achieve the same effect.
```

Comment:
To be fair the code above is incorrect (FutureProvider doesn't filter updates), so we might as well remove it and only keep the select sample